tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -15,6 +16,16 @@ def _check_status(server_response):
         if server_response.status_code not in Success_codes:
             raise ServerResponseError.from_response(server_response.content)
 
+    def get_unauthenticated_request(self, url, request_object=None):
```

Comment:
I was planning to do that in a separate checkin anyway.  There is a bunch of overlap that I think I can reduce, but it would have been noise in this change.